docs/pages/controls/list-control.mdx

```diff
@@ -0,0 +1,110 @@
+import { Page } from '../../components/layout/Page'
+import { Props, PropItem } from '../../components/Props'
+import { Link } from '../../components/markdown/Link'
+
+import Events from '../../fragments/events.mdx'
+import { CommonProps } from '../../fragments/CommonProps'
+
+export default ({ children }) => <Page>{children}</Page>
+
+# ListBox and ListBoxItem
+
+Renders Yandex.Maps API [ListBox][listbox] and [ListBoxItem][listboxitem] controls
```

Comment:
Maybe we should separate those into two pages? 🤔 My main concern is the "Props" table mixing two elements in one table, which might be confusing; Also nothing pointing only to `ListBox` events. People ask about how to attach events a lot, so missing a separate link specifically to `ListBoxItems` events can cause more questions raised in the future. What do you think?

I'm guessing that you put them on one page because one kind of doesn't make sense without the other in terms of the example. If this is your concern, then you can use the same demo, but still, have two separate pages for them.

Reply:
@gribnoysup yeah I thought that separating ListBox and ListBoxItem will confuse users and I decided to union them in one single page. To my opinion this is not the best solution. But separating them also maybe not so good. So we can separate them now, but to my mind we should think about smarter solution in the future, especially if it will confuse users